dallasnews/dmn-vendor-wrapper
# Delete header for template

## Introduction

This PR removes the nav bar from the template. 


## Jira Ticket

[Update DMN Vendor Wrapper | Gracenote Should Remove Discover Bar](https://dmndev.atlassian.net/browse/CMS-9365)


## Testing Instructions

- Make sure that you are using node version 16 - possibly 16.19
- git checkout maintenance/deleteHeader
- npm i
- npm run build
- npm run start
- Open http://localhost:8080/
- You should see this ...


![Screen Shot 2024-08-23 at 10 58 55 AM](https://github.com/user-attachments/assets/4e3fe014-d8dd-4bfc-8438-18e6e7295c50)
